waku/v2/node/peer_manager/peer_manager.nim

```diff
@@ -200,6 +200,16 @@ proc selectPeer*(pm: PeerManager, proto: string): Option[PeerInfo] =
   else:
     return none(PeerInfo)
 
+proc getPeerInfo*(pm: PeerManager, peerId: PeerId) : Option[PeerInfo] = 
```

Comment:
@jm-clius since you designed the peer manager, is there some other preferred way of doing this?

Reply:
This `peerInfo` for the `peerId` is already available in the connection handler that triggers the `credit`/`debit` function against this peer in the first place (e.g. [here](https://github.com/status-im/nim-waku/blob/master/waku/v2/protocol/waku_store/waku_store.nim#L363)). In other words, the correct way to do this would be to change the `credit`, `debit`, `sendCheque` functions to use `peerInfo` rather than `peerId`. Unless there's some reason to use `peerId` that I'm missing?

Reply:
Right, correct